api/run.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+
+from os import environ
+
+from app import create_app
+
+app = create_app()
+bind_address = "127.0.0.1"
+listen_port = "5000"
+
+if "PORT" in environ:
+    listen_port = int(environ["PORT"])
+
+if "HOST" in environ:
+    bind_address = environ["HOST"]
+
+if __name__ == "__main__":
+    app.run(debug=True, host=bind_address, port=listen_port)
```

Comment:
The debug mode could be made configurable, don't you think?

Reply:
depends. this code is not run in the
context of uWSGI, but when you run `run.py` from a shell, which is usually when you run it locally for testing.